EIPS/eip-7549.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 7549
+title: Move committee index outside Attestation
+description: Move committee index outside of the signed Attestation message
+author: dapplion (@dapplion)
+discussions-to: https://ethereum-magicians.org/t/eip-7549-move-committee-index-outside-attestation/16390
+status: Review
+type: Standards Track
+category: Core
+created: 2023-11-01
+---
+
+## Abstract
+
+Move the committee `index` field outside of the signed Attestation message to allow aggregation of equal consensus votes.
+
+## Motivation
+
+This proposal aims to make Casper FFG clients more efficient by reducing the average number of pairings needed to verify consensus rules. While all types of clients can benefit from this EIP, ZK circuits proving Casper FFG consensus are likely to have to most impact.
+
+On a beacon chain network with at least 262144 active indexes it's necessary to verify a minimum of `ceil(32*64 * 2/3) = 1366` attestations to reach a 2/3 threshold. Participants cast two votes at once: LMD GHOST vote and Casper-FFG vote. However, the Attestation message contains three elements:
+
+1. LMD GHOST vote `(beacon_block_root, slot)`. Note: includes slot in the event (block, slot) voting is adopted.
+2. FFG vote `(source, target)`
+3. Committee index `(index)`
+
+Signing over the 3rd item causes tuples of equal votes to produce different signing roots. If the committee index is moved outside of the Attestation message the minimum number of attestations to verify to reach a 2/3 threshold is reduced to `ceil(32 * 2/3) = 22` (a factor of 62).
+
+## Specification
+
+### Execution layer
+
+This requires no changes to the Execution Layer.
+
+### Consensus layer
+
+- Move `index` field from `AttestationData` to `Attestation`
+
+The full specification of the proposed change can be found in [`/specs/_features/eip7549/beacon-chain.md`](https://github.com/ethereum/consensus-specs/blob/bcead2ff59048dba859c7eb4b62389f0e9168ef8/specs/_features/eip7549/beacon-chain.md).
+
+## Rationale
+
+### Deprecation strategy
+
+The `index` field in `AttestationData` can be deprecated by:
+
+1. Removing the field 
+2. Preserving the field and setting it to be zero
+3. Changing the field type to Optional (from EIP7495 StableContainer)
```

Comment:
```suggestion
3. Changing the field type to Optional (from EIP-7495 StableContainer)
```
`EIP-N` (not `EIPN` or `EIP N`)

Reply:
When using the dash notation, a linker requested me to link the EIP. How can I do that? Using the full URL returned another linter error